src/strings.js

```diff
@@ -1,25 +1,26 @@
-function sayHello (string) {
-  // your code here
+function sayHello(string) {
+ return "Hello," + " " + string + "!";
```

Comment:
2 spaces of indentation please, theres a few others in this file that need fixing too:
```suggestion
  return "Hello," + " " + string + "!";
```